File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var searchRange = function(nums, target) {
    let arr=[];
   arr.push(nums.indexOf(target))
   arr.push(nums.lastIndexOf(target))
   return arr
};

File: 0034-find-first-and-last-position-of-element-in-sorted-array/0034-find-first-and-last-position-of-element-in-sorted-array.cs
public class Solution {
    public int[] SearchRange(int[] nums, int target) {
    int[] arr1= new int[2];
     
        int index=Array.IndexOf(nums,target);
        int index2=Array.LastIndexOf(nums,target);
        arr1[0]=index;
      arr1[1]=index2;
      return arr1;
    }
}